.github/workflows/grid_client_tests.yml

```diff
@@ -2,32 +2,34 @@ name: Grid Client Tests
 
 on:
   schedule:
-    - cron: "0 0 * * *"
+    - cron: 0 0 * * *
   workflow_dispatch:
     inputs:
       network:
         type: choice
-        description: 'which network to run tests on'
+        description: "Which network to run tests on"
         required: true
-        default: 'dev'
+        default: "dev"
         options: 
-        - 'dev'
-        - 'qa'
-        - 'test'
-        - 'main'
-
+        - "dev"
+        - "qa"
+        - "test"
+        - "main"
```

Comment:
Can we add custom option? We'll also need to provide all urls.

Reply:
The workflow is only for the default setups though, and any custom options would need to be manually run. These tests are meant to run in basic environments.